SWARAM-UI-Documentation/components.css

```diff
@@ -0,0 +1,512 @@
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons+Outlined");
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons");
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap');
+
+:root{
+    --primary:#aa3a3a;
+    --secondary:#C4C6A6;
+    --padding-1:0.5rem;
+    --padding-2:0.8rem;
+    --padding-3:0.4rem;
+    --font-size-text:1.1rem;
+    --font-size-small:0.8rem;
+    --font-size-title:01.4rem;
+    --font-family:Roboto;
+    --margin-right:0.6rem;
+}
+
+body{
+    margin:0;
+}
+
+.bg-primary{
+    background-color: var(--primary);
+    color:white;
+}
+
+.text-color-primary{
+    color:var(--primary)
+}
+
+.flex-horizontal{
+    display: flex;
+}
+
+/** Avatar **/
+
+.avatar-container{
+    position: relative;
+}
+
+.img-size-medium{
+    width:5.5rem;
+    height: 5.5rem;
+}
+
+.img-size-large{
+    width:7.5rem;
+    height: 7.5rem;
+}
+
+.img-size-small{
+    width:3.5rem;
+    height: 3.5rem;
+}
+.img-size-xs{
+    width:2rem;
+    height: 2rem;
+}
+
+/** Alerts **/
+
+.alert{
+    padding:var(--padding-2) 0rem;
+    padding-left:1rem;
+    margin-bottom: 1rem;
+    font-size:var(--font-size-text);
+    position: relative;  
+    border-radius:0.3rem
+}
+
+.alert-success{
+    background-color:#A7F3D0;
+    color:#0f5132;
+}
+
+.alert-error{
+    background-color:#FECACA;
+    color:#842029;
+}
+
+.alert-warning{
+    background-color: #f3e096;
+    color:#664d03;
+}
+
+/** Badges **/
+
+.badge{  
+    position:absolute;
```

Comment:
Is it a good idea to position  : absolute a component?

Reply:
Yeah I want the badge to placed on top of another component.So I have made it position absolute.